Controller/ContentTypeController.php

```diff
@@ -350,4 +360,23 @@ public function deleteContentTypeAction(Request $request, $contentTypeId)
 
         return $this->redirectToRouteAfterFormPost('admin_contenttypeView', ['contentTypeId' => $contentTypeId]);
     }
+
+    /**
+     * Return the highest prioritized language that $contentType is translated to.
+     * If there is no translation for a prioritized language, return $contentType's main language.
+     *
+     * @param ContentType $contentType Content type (or content type draft)
+     *
+     * @return string Language code
+     */
+    private function getPrioritizedLanguage(ContentType $contentType)
+    {
+        foreach ($this->prioritizedLanguages as $prioritizedLanguage) {
+            if (in_array($prioritizedLanguage, array_keys($contentType->names))) {
```

Comment:
Come to think about, it could have been `if (isset($contentType->names[$prioritizedLanguage]))` more readable imo